packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx

```diff
@@ -22,23 +19,29 @@ export default {
         {storyFn()}
       </div>
     ),
-  ],
+  ]}
+/>
+
+# Form Title
+
+The Form Title component is used for forms, mostly on subdomains of amsterdam.nl. Forms don't have a homepage and can therefore **not** have a `title` in the `<Header>` component.
 
-  parameters: {
-    notes: {
-      markdown: Readme,
-    },
-  },
-}
+[Design System](https://designsystem.amsterdam.nl/7awj1hc9f/p/537bb7-title-component)
 
-export const Default = () => (
-  <ThemeProvider>
+**Component status: stable**
+
+## Default
+
+<Preview>
+  <Story name="Default">
     <FormTitle>This is the title of the FormTitle component</FormTitle>
-  </ThemeProvider>
-)
+  </Story>
+</Preview>
+
+## In context
 
-export const InContext = () => (
-  <ThemeProvider>
+<Preview>
+  <Story name="In context">
     <Header homeLink="" tall />
     <FormTitle>This is the title of the FormTitle component</FormTitle>
     <Card maxWidth={500} backgroundColor="level2" shadow>
```

Comment:
Somehow FormTitle and Card are not shown in Netlify preview in this Story

![image](https://user-images.githubusercontent.com/7781865/81424628-8a327900-9156-11ea-849d-6af0cf468879.png)

Reply:
Yeah, haven't been able to figure this one out either. Looks like in all cases only the first component is rendered.

Reply:
![image](https://user-images.githubusercontent.com/7781865/81588908-4d19f100-93b9-11ea-9a01-1af57c207e12.png)
Looks good now! Great